### ISSUE TITLE
App crash

### ISSUE THREAD

**Issue by ingsaurabh** [AppLovin/AppLovin-MAX-SDK-Android#180]
Kindly make the sdk number specific instead of +, I uploaded the release build yesterday and now getting this crash

Currently using

`implementation 'com.applovin:applovin-sdk:+'`

```
Fatal Exception: java.lang.IllegalStateException
Player is accessed on the wrong thread. Current thread: 'AsyncTask #217' Expected thread: 'main' See https://exoplayer.dev/issues/player-accessed-on-wrong-thread
com.google.android.exoplayer2.SimpleExoPlayer.verifyApplicationThread (SimpleExoPlayer.java:2114)

com.google.android.exoplayer2.SimpleExoPlayer.getCurrentPosition (SimpleExoPlayer.java:1774)
com.applovin.impl.adview.activity.b.e.C (e.java:2)
com.applovin.impl.adview.activity.b.e.l (e.java)
com.applovin.impl.adview.activity.b.e.h (e.java:11)
com.applovin.impl.adview.activity.b.a$4$1.run (a.java:11)
com.applovin.impl.sdk.e.z.run (z.java:2)
java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1167)
```

**Comment by RunningTheSnail** [comment]
I also had the same problem

**Comment by thomasmso** [comment]
Hi, we're aware of the issue and will be fixed in an SDK release this weekend.

**Comment by ingsaurabh** [comment]
Thanks for quick response, kindly make the new release and mediation adapters version specific rather than this generic, will help in reverting the build if such crashes arise in future

**Comment by RunningTheSnail** [comment]
> Thanks for quick response, kindly make the new release and mediation adapters version specific rather than this generic, will help in reverting the build if such crashes arise in future

Which version should you use without this problem?

**Comment by ingsaurabh** [comment]
> > Thanks for quick response, kindly make the new release and mediation adapters version specific rather than this generic, will help in reverting the build if such crashes arise in future
> 
> Which version should you use without this problem?

Applovin SDK doesnt specify version number, so don’t know, had I know the last working version I would have reverted to that one, We got this crash for 80 users till now, before this crash we had 99.5% above crash free users now this has been reduced to 98.5 and I guess this will go down further

**Comment by santoshbagadi** [comment]
Hi @ingsaurabh , you can always lock the version to a specific version you need by replacing the `+` to a version in our docs here: https://dash.applovin.com/documentation/mediation/android/changelog

`implementation 'com.applovin:applovin-sdk:11.1.2'`

You can also check the version integrated in your build by running the builds with [verbose logging enabled](https://monetization-support.applovin.com/hc/en-us/articles/236114328-How-do-I-enable-verbose-logging-in-AppLovin-s-SDK-) to see what version of our SDK is integrated. The logs should look something like this: 
```
    ========== MAX ==========
    SDK Version - x.y.z
    Plugin Version - Max-Unity-x.y.z
    Ad Review Version - x.y.z
```

**Comment by thomasmso** [comment]
We just released Android SDK v11.1.3 which addresses the ExoPlayer crash.

**Comment by ingsaurabh** [comment]
Hi @santoshbagadi latest version in change log is showing 11.1.3, mentioning its been fixed can you confirm the same
